alf/examples/sac_bipedal_walker.gin

```diff
@@ -46,5 +44,5 @@ TrainerConfig.num_checkpoints=5
 TrainerConfig.evaluate=False
 TrainerConfig.debug_summaries=True
 TrainerConfig.summarize_grads_and_vars=True
-TrainerConfig.summary_interval=1
-TrainerConfig.replay_buffer_length=1000000
+TrainerConfig.summary_interval=100
+TrainerConfig.replay_buffer_length=100000
```

Comment:
Since now the parameters are changed, it is better to update the png curves to maintain the correspondence